scripts/kill-ports.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Check for required commands
+for cmd in lsof rg awk xargs; do
```

Comment:
Do we install all of these tools using nix?

Reply:
Oops. this script wasn't supposed to be in here. Will rebase. devimint has been improved to fix this.
https://github.com/fedimint/fedimint/pull/2873